src/legacy-components/BoxShape.ts

```diff
@@ -24,6 +24,7 @@ export const BoxShape: EcsType<BoxShape> = {
     newValue.isPointerBlocker = reader.bb.readByte() === 1
     newValue.withCollisions = reader.bb.readByte() === 1
     const length = reader.bb.readUint16()
+    newValue.uvs = []
```

Comment:
can we remove the as any if we initalize the values inside the obj ? 
or how reader works this is no possible ? 
```ts

const newValue: BoxShape = {
 visible: reader.bb.readByte() === 1,
  ..etc,
  uvs: Array.from({ length }, () => reader.bb.readFloat32()),
}
```